package.json

```diff
@@ -172,9 +222,10 @@
     "eslint-plugin-react": "6.4.1",
     "eslint-plugin-wpcalypso": "3.0.2",
     "glob": "7.0.3",
+    "husky": "0.13.3",
     "lodash-deep": "1.5.3",
     "md5-file": "3.1.0",
-    "mixedindentlint": "1.1.1",
+    "mixedindentlint": "DanReyLop/mixedindentlint#add/cli-globs",
```

Comment:
Waiting for https://github.com/sirbrillig/mixedindentlint/pull/4 to be merged and released.